cmake/ecbuild_check_os.cmake

```diff
@@ -424,9 +424,15 @@ if( WIN32 )
 
   set( EC_OS_NAME "windows" )
 
+  find_program( BASH NAMES bash )
```

Comment:
The variable for a program is usually post fixed with _EXE or _BINARY.
I suggest using BASH_EXE.

Reply:
Please add a DOC entry to the find_program specifying for what this will be used.
e.g.  `find_program( BASH_EXE NAMES bash DOC "Used under Windows for fixing symlinks and running unit tests" )`

Reply:
Both done